coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -673,7 +672,7 @@ describe('Appoint New Director tests', () => {
     setTimeout(() => {
       expect(vm.$el.querySelectorAll('.v-messages')[0].textContent).toBe('')
       done()
-    }, 200)
+    }, 250)
```

Comment:
I know i've asked before and i'm probably a pain in the butt here, but what defines the time required in these? Will Async and await not do the trick in these cases?

Reply:
We have not been able to get these working with async/await or flushPromises. I dunno why. (Guess: there's debouncing happening on the input?)

When these tests fail when I run them locally, increasing the timeouts usually works.

It bugs me but I had a quick look and didn't find a solution. You're welcome to try :)

Reply:
Sounds good, working tests are whats important, so we can always revisit this when we have time!